app/controllers/reset_passwords_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ResetPasswordsController < ApplicationController
+  skip_before_action :authenticate_user
+  before_action :ensure_no_user
+  before_action :pundit_authorize_user
```

Comment:
I believe so. Especially that resetting password is not resource by itself, so it makes no sense to create policy for it. Unfortunately sometimes there is very thin line between 'should' and 'should not' when it comes to using authorization :wink: